### ISSUE TITLE
Implement a "Select All" menu item

### ISSUE THREAD

**Issue by mat-m** [nextcloud/mail#7880]
### Is your feature request related to a problem? Please describe.

I can't select all items in a folder/mailbox to apply the same action to them, typîcally move or delete.

### Describe the solution you'd like

After selecting one element in the messages list, the "Actions on selection" 3-dot menu at the top could benefit from a "Select all" entry, so you can then apply any action on all the emails you have.

### Describe alternatives you've considered

There no keybaord  shortcut to enable selection like the "S" does for starring.

### Additional context

It'a quick win for multiple mails handling.  #7276 has more in-depth desires which would be handy.

**Comment by guettli** [comment]
I have the vague feeling, that nobody is using the mail feature of Nextcloud. A mail user agenet which does not support these steps is unusable for me:

1. search
1. select all mails which where found by the search
1. Do a bulk action on the select mails.

Step 1 and 3 are implemented.

But Step 2 is missing. 

I could click on 100 mails ... yes I could do it .... but I don't want to.

**Comment by mat-m** [comment]
> For the use case of deletion this is a duplicate of [#4286 (comment)](https://github.com/nextcloud/mail/issues/4286#issuecomment-754065575)

No. on #4286, user wants a dedicated button only for Trash folder.
Here I want a "Select all" on every mailbox, and be able to choose my action, whether it's  move, delete, mark as spam, favorite, forward as attachment, ...

**Comment by turboyz** [comment]
Same here, I'd need this feature so badly. I need to manage professionally hundreds of emails.

In Gmail for example they implemented a selection mode similar to the file selection for folder. 

CTRL+left_click for selecting the single email 
SHIFT+left_click for selecting all the emails between your first selection and your last selection.
